### DISCUSSION TITLE
can't find MidiFileGenerator.GenerateMidiFile

### DISCUSSION THREAD

**Discussion by yibo7** [CoderLine/alphaTab#764] [Q&A] [answered]
AlphaTab 1.3   can't find MidiFileGenerator.GenerateMidiFile(score);

**Comment by Danielku15** [comment, marked as answer]
Not sure what you mean by this. 
1. Version 1.3 is still in active development and not meant for use at this point. Better stick to the stable, officially released versions (not pre-release)
2. In none of the official releases since 1.0 there was a `MidiFileGenerator.GenerateMidiFile`. To generate a midi model using low level APIs the usage is like: https://alphatab.net/docs/guides/lowlevel-apis#generating-midi-files-via-midifilegenerator

**Reply by yibo7** [reply]
thank you
